javascript-modal/main.js

```diff
@@ -0,0 +1,20 @@
+let toggleModal = false;
+const $modalButton = document.querySelector('.modal-button');
+const $noButton = document.querySelector('.no-button');
+const $modalOverlay = document.querySelector('#modal');
+
+$modalButton.addEventListener('click', function (event) {
+  toggleModal = !toggleModal;
+  console.log(toggleModal);
```

Comment:
Remove all console logs before submitting! They are purely for testing / debugging purposes. Unless we specifically ask to include them in the instructions.